patches/chrome-browser-push_messaging-push_messaging_service_impl.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/push_messaging/push_messaging_service_impl.cc b/chrome/browser/push_messaging/push_messaging_service_impl.cc
+index eeec06df57636d7452669884a5a249680ce68def..d08b9d01daf8413434ec5699c79f0be7ddce1bd0 100644
+--- a/chrome/browser/push_messaging/push_messaging_service_impl.cc
++++ b/chrome/browser/push_messaging/push_messaging_service_impl.cc
+@@ -171,7 +171,7 @@ void PushMessagingServiceImpl::InitializeForProfile(Profile* profile) {
+ 
+   PushMessagingServiceImpl* push_service =
+       PushMessagingServiceFactory::GetForProfile(profile);
+-  push_service->IncreasePushSubscriptionCount(count, false /* is_pending */);
++  if (push_service) { push_service->IncreasePushSubscriptionCount(count, false /* is_pending */); }
```

Comment:
`GetForProfile` can return NULL, upstream issue here: https://bugs.chromium.org/p/chromium/issues/detail?id=999665

Reply:
Is this the only place where the check is needed? I'm concerned about https://bugs.chromium.org/p/chromium/issues/detail?id=999665#c2

Reply:
I filed that issue. For Brave `IsInstanceIDEnabled` returns false since field trials are disabled. I can update the patch to change `IsInstanceIDEnabled` to return `true` instead?

Reply:
I'm ok with this patch, given that this check is enough. I'm just not sure if we will not hit any other null `push_service` pointers.

Reply:
cc: @bridiver 

Looking at the chromium code looks like other `push_service` pointers are covered -

https://cs.chromium.org/chromium/src/content/browser/push_messaging/push_messaging_context.cc?l=32
https://cs.chromium.org/chromium/src/content/browser/push_messaging/push_messaging_context.cc?l=34
https://cs.chromium.org/chromium/src/content/browser/push_messaging/push_messaging_context.cc?l=43
https://cs.chromium.org/chromium/src/content/browser/push_messaging/push_messaging_manager.cc?l=423